cardano_node_tests/tests/test_scripts.py

```diff
@@ -1169,6 +1177,126 @@ def test_script_before(
         dbsync_utils.check_tx(cluster_obj=cluster, tx_raw_output=tx_out_to)
         dbsync_utils.check_tx(cluster_obj=cluster, tx_raw_output=tx_out_from)
 
+    @allure.link(helpers.get_vcs_link())
+    @common.PARAM_USE_BUILD_CMD
+    def test_missing_validity_script_after(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addrs: List[clusterlib.AddressRecord],
+        use_build_cmd: bool,
+    ):
+        """Check that it is NOT possible to spend from script address after given slot.
+
+        The transaction validity interval is not specified.
+        """
+        temp_template = f"{common.get_test_id(cluster)}_{use_build_cmd}"
+
+        payment_vkey_files = [p.vkey_file for p in payment_addrs]
+        payment_skey_files = [p.skey_file for p in payment_addrs]
+
+        # create multisig script
+        multisig_script = cluster.g_transaction.build_multisig_script(
+            script_name=temp_template,
+            script_type_arg=clusterlib.MultiSigTypeArgs.ALL,
+            payment_vkey_files=payment_vkey_files,
+            slot=100,
+            slot_type_arg=clusterlib.MultiSlotTypeArgs.AFTER,
+        )
+
+        # create script address
+        script_address = cluster.g_address.gen_payment_addr(
+            addr_name=temp_template, payment_script_file=multisig_script
+        )
+
+        # send funds to script address
+        multisig_tx(
+            cluster_obj=cluster,
+            temp_template=f"{temp_template}_to",
+            src_address=payment_addrs[0].address,
+            dst_address=script_address,
+            amount=4_000_000,
+            payment_skey_files=[payment_skey_files[0]],
+            use_build_cmd=use_build_cmd,
+        )
+
+        # send funds from script address - missing required validity interval
+        with pytest.raises(clusterlib.CLIError) as excinfo:
+            multisig_tx(
+                cluster_obj=cluster,
+                temp_template=f"{temp_template}_from",
+                src_address=script_address,
+                dst_address=payment_addrs[0].address,
+                amount=2_000_000,
+                payment_skey_files=payment_skey_files,
+                multisig_script=multisig_script,
+                invalid_before=None,  # missing required validity interval
+                invalid_hereafter=None,
+                use_build_cmd=use_build_cmd,
+            )
+        err_str = str(excinfo.value)
+        assert "ScriptWitnessNotValidatingUTXOW" in err_str, err_str
+
+    @allure.link(helpers.get_vcs_link())
+    @common.PARAM_USE_BUILD_CMD
+    def test_missing_validity_script_before(
```

Comment:
optional: maybe we can join the 2 new tests, because the code is basically the same, also same error.